### PR TITLE
Project cleanup, update dependencies, fix 1.16 compatibility

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: src/main/java/de/oppermann/bastian/safetrade/listener/InventoryDragListener.java [hunk 1]
```diff
@@ -27,9 +26,7 @@
      */
     @EventHandler
     public void onInventoryDrag(final InventoryDragEvent event) {
-        if (event.getWhoClicked().getOpenInventory() == null) {
-            return;
-        }
+        event.getWhoClicked().getOpenInventory();
```
Comment (Bastian) [inline src/main/java/de/oppermann/bastian/safetrade/listener/InventoryDragListener.java:29]: Why did you remove this check? Just calling the getter seems pretty useless?

File: src/main/java/de/oppermann/bastian/safetrade/listener/InventoryClickListener.java [hunk 2]
```diff
@@ -181,10 +175,13 @@ private Inventory getClickedInventory(InventoryClickEvent event) {
         // https://hub.spigotmc.org/stash/projects/SPIGOT/repos/spigot/browse/Bukkit-Patches/0009-InventoryClickEvent-getClickedInventory.patch
         if (event.getRawSlot() < 0) {
             return null;
-        } else if (view.getTopInventory() != null && event.getRawSlot() < view.getTopInventory().getSize()) {
-            return view.getTopInventory();
         } else {
-            return view.getBottomInventory();
+            view.getTopInventory();
```
Comment (Bastian) [inline src/main/java/de/oppermann/bastian/safetrade/listener/InventoryClickListener.java:179]: This getter call seems unnecessary. Besides that: As you dropped Bukkit support by using `target.spigot().sendMessage(generateTextComponents());` anyway, we can just use `InventoryClickEvent#getClickedInventory` and drop the whole method.

**Review by Bastian — COMMENTED** [review]
Thank you very much! Can you take a look at my two review comments?

File: src/main/java/de/oppermann/bastian/safetrade/listener/InventoryDragListener.java (hunk 1 above)
Comment (sgdc3) [reply src/main/java/de/oppermann/bastian/safetrade/listener/InventoryDragListener.java:29]: oh, missed that, i intendes to remove the entire check as the getter is marked as @NonNull

**Comment by Bastian** [comment]
Looks good to me now, tyvm! 🚀

**Review by Bastian — APPROVED** [review]
